### DISCUSSION TITLE
forge-pull fails on a specific repository

### DISCUSSION THREAD

**Discussion by adamist521** [magit/forge#349] [Support] [answered]
I have specific repository which fails with attached error. (can not `forge-pull` )

I have repositories that has same privileges with this repository which successes to `forge-pull`.

Non of the following fixed my issue...
- execute `forge-reset-database` and `forge-pull` again
- execute `ghub-clear-cache` and `forge-pull` again
- clone repository in another location and execute `forge-pull`

Environments
- MacOS 10.15.6
- Emacs 27.1 (uses doom)
- Github Enterprise Server 2.21.14 

Any ideas how I could solve this issue?

```
error in process filter: peculiar error: ((message . "Something went wrong while executing your query. Please include `ac45ca2a-0863-4c28-af17-9bdb35293f97` when reporting this issue."))
```

debug on error (truncated long lines with *. GHE url hidden by *.)
```
Debugger entered--Lisp error: (ghub-graphql-error ((message . "Something went wrong while executing your query. P...")))
  signal(ghub-graphql-error (((message . "Something went wrong while executing your query. P..."))))
  ghub--signal-error((ghub-graphql-error ((message . "Something went wrong while executing your query. P..."))))
  ghub--graphql-handle-response((:peer (:certificates ((:version 3 :serial-number "53:60:3  ****************
  apply(ghub--graphql-handle-response ((:peer (:certificates ((:version 3 :serial-number "53:6  *************
  url-http-activate-callback()
  url-http-chunked-encoding-after-change-function(1065 1228 163)
  url-http-wait-for-headers-change-function(1 1254 1253)
  url-http-generic-filter(#<process ***ghe_address***> "HTTP/1.1 200 OK\15\nServer: GitHub.com\15\nDate: Tue, 13...")

```

**Comment by tarsius** [comment, marked as answer]
> Something went wrong while executing your query. Please include `ac45ca2a-0863-4c28-af17-9bdb35293f97` when reporting this issue.

This error message comes from the Github server. You could try to ask the maintainer of this Github instance to provide the error log that is indexed by that hash. Without that it's "something went wrong and we have no way of knowing what", I'm afraid.

**Reply by adamist521** [reply]
Thanks for response!

OK, I'll try asking the maintainer.
